Doc/howto/argparse.rst

```diff
@@ -841,6 +841,69 @@ translated messages.
 
 To translate your own strings in the :mod:`argparse` output, use :mod:`gettext`.
 
+Custom type converters
+======================
+
+The :mod:`argparse` module allows you to specify custom type converters for
+your command-line arguments. This allows you to modify user input before it's
+stored in the :class:`argparse.Namespace`. This can be useful when you need to
+pre-process the input before it is used in your program.
```

Comment:
```suggestion
The :mod:`argparse` module allows you to specify custom type converters for
your command-line arguments. This allows you to modify user input before it is
stored in the :class:`argparse.Namespace`. This can be useful when you need to
pre-process the input before it is used in your program.
```

Reply:
I'll probably stick with the tone that I have here because the tutorial is pretty casual and regularly uses "it's" etc.